ros/src/shared_utils/include/shared_utils/params.py

```diff
@@ -60,13 +60,10 @@ def Get(self): #, default_override = None):
             else:
                 raise ("Unknown scope: {}".format(self.scope))
             self.saved_value = value
-
+            
```

Comment:
I've added this first logdebut call back -- since it will only log the parameter once. But I've removed the log statement for the ongoing case.